tox.ini

```diff
@@ -3,35 +3,39 @@
 [tox]
 skipsdist=True
 skip_missing_interpreters = True
-envlist = lint, static, unit
+envlist = lint-v1, lint-v2, static-v1, static-v2, unit-v1, unit-v2
```

Comment:
I wonder why the split to v1, v2 is necessary here.
Shouldn't `static`, `lint` and `unit` just work with `lib_path = {toxinidir}/lib/charms/tls_certificates_interface`?

Reply:
The intent was to make the CI specific to the change we are making. For example, if I'm making a change to the lib v1, I want to test only test and I don't want my PR to be prevented because of a static analysis issue on v2.